tests/test_toolforge.py

```
@@ -1,166 +1,147 @@
-    response = client.post(
-        "/api/toolforge/jobs/v1/tool/test-tool/jobs/",
-        json=job_config,
-        headers={"X-API-KEY": "invalid-key"}
-    )
+    response = client.delete("/api/toolforge/jobs/v1/tool/test-tool/jobs/job1")
     assert response.status_code == 401
+    assert response.json()["detail"] == "Unauthorized"
+    app.dependency_overrides = {}
+
+# Test for get_tool_jobs which does not require authentication
+def test_get_tool_jobs_no_auth_needed():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)